<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="dropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {	
	
	background-color: rgb(95, 86, 116);
	color: rgb(220, 220, 220);
	
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>661</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(43, 22, 74);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Hệ thống đếm và phân loại &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;sản phẩm phân bón&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>260</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;APP&lt;/strong&gt; DESCRIPTION</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>360</y>
         <width>561</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>390</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>470</y>
         <width>621</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Created&lt;/span&gt;: CTUT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/logo/phan_bon_cm.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/logo/Logo_ctuet.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/logo/ctu.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/logo/kgst.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_description</zorder>
      <zorder>progressBar</zorder>
      <zorder>label_loading</zorder>
      <zorder>label_credits</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>label_title</zorder>
      <zorder>label</zorder>
      <zorder>label_4</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
